.github/workflows/c.yaml

```diff
@@ -252,7 +268,8 @@ jobs:
   windows_build:
     name: Windows build
     needs: windows_setup
-    uses: openslide/openslide-winbuild/.github/workflows/windows.yml@main
+    uses: jcupitt/openslide-winbuild/.github/workflows/windows.yml@main
+    # uses: openslide/openslide-winbuild/.github/workflows/windows.yml@main
```

Comment:
Any reason we can't use the upstream one?

Reply:
Now it's merged, sure. I'll flip it over.